### PR TITLE
Move all portlet related to p.a.portlets

### PR DESCRIPTION
**Pull request by jensens**
Untangle plone.app.layout/plone.app.portlets. First step towards Portlets-as-an-Addon.
together with https://github.com/plone/plone.app.portlets/pull/160

### CODE REVIEW

**Comment by ale-rt** [comment]
At a first glance it looks good to me. Give me a little more time to check it please!

File: plone/app/layout/viewlets/common.py [hunk 1]
```diff
@@ -32,14 +31,17 @@
...
-
-try:
-    from html import escape
-except ImportError:
-    from cgi import escape
+zope.deferredimport.initialize()
+zope.deferredimport.deprecated(
+    "Import from plone.app.portlets.browser.viewlets instead",
+    ManagePortletsFallbackViewlet='plone.app.portlets.browser.viewlets:baaz.ManagePortletsFallbackViewlet',
+    FooterViewlet='plone.app.portlets.browser.viewlets:baaz.FooterViewlet',
```
Comment (ale-rt) [inline plone/app/layout/viewlets/common.py:43]: Is this `baaz` wanted?

Comment (jensens) [reply plone/app/layout/viewlets/common.py:43]: ops

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by ale-rt — APPROVED** [review]

**Pull request merged**